### ISSUE TITLE
[Bug] Issue on carbondesignsystem.com with firefox mobile

### ISSUE THREAD

**Issue by atomspring** [carbon-design-system/carbon#11498]
When on a mobile device, the menu partially pokes out for carbondesignsystem.com when using Firefox. This causes people to worry that this great quality toolkit isn't fully compatible with the three browsers people think of: Chrome, Firefox, Safari.

**Comment by atomspring** [comment]
Hey Josh, thanks for responding. I've attached a screenshot, you can see the menu poking out on the side. Menu entries are clickable, even in the "closed" state.

This is a Google pixel 3 running Android 12, Firefox mobile v100. This issue doesn't appear when browsing with chromium.

~A

-------- Original Message --------

**Comment by joshblack** [comment]
Hi @atomspring! Unfortunately, I'm unable to see the screenshot that you're describing. Is it possible that it got lost by responding via email? Wasn't sure how that flow works and if that was the source of the problem or not, feel free to correct me if I'm wrong here!

**Comment by atomspring** [comment]
Ah shoot, ok. I'm reattaching via GitHub, didn't realize attachments were limited by email.
![Screenshot_20220527-112026](https://user-images.githubusercontent.com/9324477/171309271-c8a0607d-ca9f-4c22-942e-459c3eeb474c.png)

**Comment by tw15egan** [comment]
I'm unable to reproduce running Firefox on my iPhone 13 Pro Max running iOS 15.5. Also checked Safari and Chrome and didn't see any issues. I tried running the Dev Tool emulator for an Android phone on Firefox Desktop, but I still can't replicate the issue. Can you check and see if this has been resolved? If not, can you provide what model phone/software you are running? 

![IMG_77E699890E24-1](https://user-images.githubusercontent.com/11928039/173429966-ee77285c-e9a3-4a4e-8670-2861c93ae6f6.jpeg)

**Comment by atomspring** [comment]
Oh didn't realize this has been responded to, sorry about that. 

Issue still exists with default install of firefox from google's play store, no add-ons or changed settings (v101.2.0 as of this writing).

Device is a pixel 3 running Android 12.

I've attached a screenshot of my browser dimensions, maybe that helps?
![Screenshot_20220628-111636](https://user-images.githubusercontent.com/9324477/176256291-36b0d6c9-85d8-47a1-9068-41bfe409cc0e.png)

**Comment by tw15egan** [comment]
@carbon-design-system/developers Can someone with an Android device confirm this issue?

**Comment by atomspring** [comment]
Can this be reopened?

**Comment by atomspring** [comment]
Welp. Guess I'm not using this framework. Good luck guys!

**Comment by atomspring** [comment]
Figured out this issue, accidentally. Don't size for my device was set to "Large" vs the stock "default", on resetting back to "default", carbon framework displays correctly! Is there a way to programmatically detect this? I'm sure the browser is *supposed* to handle this properly..

**Comment by dakahn** [comment]
@tw15egan This should be tagged as an accessibility issue since it effects users of increased font size. I was unable to repro with max font size on iOS using Safari. Each browser is going to handle dynamic fonts a little differently though

**Comment by atomspring** [comment]
@dakahn just wondering, if you install Firefox mobile on iOS, does it also suffer the same issue I did on Android?

**Comment by ljcarot** [comment]
@atomspring I installed FF on my iOS and it appears correctly.  I'm on iPhone XR iOS 15.6.1.  I have increased font size too and it still displays as expected.

**Comment by atomspring** [comment]
this happens on my Pixel 3 (Android 12) using Firefox 101.2. I have no add-ons, only default settings.

~A

-------- Original Message --------
